src/redux/userSlice.tsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit';
+interface UserState {
```

Comment:
It's time to create src/types folder with files like: user.ts, tickets.ts, etc. where we'll store all common types

Reply:
types created, interfaces for users moved to user.ts